app/full_text_search.py

```diff
@@ -37,6 +37,22 @@ def make_or_regexp(query_txt):
     regexp = '|'.join([re.escape(word) for word in query_words if word])
     return re.compile(regexp, re.I)
 
+def create_sort_expression(**kwargs):
+    """
+    Creates sortExpressions for ranking.
```

Comment:
s/sortExpressions/SortExpression's/
(If it's "SortExpressions", it looks like the class name is "SortExpressions". So I use ['s] instead. Not sure if it's a common way.)